packages/e2e/src/page-objects/messages.page.ts

```diff
@@ -19,23 +19,220 @@ class MessagesPage extends Page {
   }
 
-  public get notSendMessageIcon(): ChainablePromiseElement<
+  public get sendMessageButton(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="thread-details-text-area-send-button"]')
+  }
+
+  public get messageNotSentIcon(): ChainablePromiseElement<
     Promise<WebdriverIO.Element>
   > {
     return $('[data-testid="message-bubble-not-send-icon"]')
   }
 
-  public get notSendThreadIcon(): ChainablePromiseElement<
+  public get messageNotSentThreadIcon(): ChainablePromiseElement<
     Promise<WebdriverIO.Element>
   > {
     return $('[data-testid="thread-not-send-message-icon"]')
   }
+
+  public get deleteButtonOnThreadDetailScreen(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="icon-Delete"]')
+  }
+
+  public get addContactButtonOnThreadDetailScreen(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="icon-NewContact"]')
+  }
+
+  public get closeButtonOnThreadDetailScreen(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="icon-Close"]')
+  }
+
+  public get markUnreadButtonOnThreadDetailScreen(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="icon-Close"]')
+  }
+
+  public get sendingStatus(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="dot"]')
+  }
+
+  public get messageContainer(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="message-bubble-container"]')
+  }
+
+  public get containerMessageContent(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="message-bubble-message-content"]')
+  }
+
+  async hoverOverMessage() {
+    await this.containerMessageContent.waitForDisplayed({ timeout: 9000 })
+    await this.containerMessageContent.moveTo()
+  }
+
+  public get messageOptionsDropdownList(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="message-bubble-dropdown-action-button"]')
+  }
+
+  //
+  public get optionsButtonSingleMessage(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return this.messageContainer.$('[data-testid="icon-More"]')
+  }
+
+  async clickSingleMessageOptionsButton() {
+    await this.optionsButtonSingleMessage.waitForClickable()
+    await this.optionsButtonSingleMessage.click()
+  }
+
+  public get threadScreenEmptyList(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="messages-empty-thread-list-state"]')
+  }
+
+  public get threadScreenThreadDropdownOptions(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
+  > {
+    return $('[data-testid="dropdown"]')
+  }
+
+  public get deleteOnThreadDropdownOptions(): ChainablePromiseElement<
```

Comment:
this is a list or a element of the options list? 🤔 
```suggestion
  public get deleteOnThreadDropdownOption(): ChainablePromiseElement<
```